src/com/hillel/Main.java

```diff
@@ -1,8 +0,0 @@
-package com.hillel;
```

Comment:
Why did you delete this class? That's why you have merge conflict. Restore this class from master (on your branch check master and Merge into Current)